nextjs-logs-manager/app/utils/handleTxt.ts

```
@@ -0,0 +1,33 @@
+    });
+  };
+
+
+ export  const parseLogFile = (content: string): ILogs[] | null  => {
+
+    try {
+        const logPattern = /\[(.*?)\] \[(.*?)\] (.*?)\: (.*?)(?=\s*\[|\s*$)/g;
+        const logs: ILogs[] = [];
+        let match;
+    
+        while ((match = logPattern.exec(content)) !== null) {
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)